Cargo.toml

```
@@ -12,10 +13,9 @@ build       = "build.rs"
 bitflags    = "1.3.2"
 lazy_static = "1.4.0"
 libc        = "0.2.103"
-nix         = "0.23.0"
+nix         = "0.24.0"
```

nix `0.24.1` is out now, and worth uptaking. You can also reduce build times slightly by limiting how much of nix you build.

```suggestion
nix         = { version = "0.24.1", default-features = false, features = ["net"] }
```